### PR TITLE
[INLONG-6961][Dashboard][Manager] Support for configing non-InlongMsg formats

### LINKED ISSUE #6961: [Feature][Dashboad][Manager] Support for handling non-InlongMsg formats
### Description

All data types used chosen will be packaged by InLongMsg default. However, when the user reports data, it may happen that the raw format is used.

企业微信截图_1e707384-a0ea-4b0b-b8a7-d35a38b7c629

企业微信截图_08c9dccb-cc80-4476-8e5f-41a291edde78

企业微信截图_259d8af5-b77c-4444-a5e3-93e27fd646b7

企业微信截图_fde4ee40-a53a-49ce-8dba-7b223ab91c40

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by featzhang**
### Prepare a Pull Request

**[INLONG-6961][Dashboard][Manager] Support for configing non-InlongMsg formats**

- Fixes #6961 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/InlongStreamInfo.java [hunk 1]
```diff
@@ -63,6 +63,9 @@ public class InlongStreamInfo extends BaseInlongStream {
     @ApiModelProperty(value = "Data type, including: TEXT, KV, etc.")
     private String dataType;
 
+    @ApiModelProperty(value = "Whether the message body wrapped with InlongMsg, 0: no, 1:yes (as default).")
```
Comment (healchow) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/InlongStreamInfo.java:66]:
```suggestion
    @ApiModelProperty(value = "Whether the message body wrapped with InlongMsg, 0: no, 1: yes (as default)")
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java [hunk 2]
```diff
@@ -118,9 +116,8 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
                 kafkaSource.setSerializationType(sourceInfo.getSerializationType());
             }
 
-            // CSV: InLong message type whose message body is raw CSV
-            // Raw-CSV: messages are separated by a specific separator
-            kafkaSource.setWrapWithInlongMsg(streamInfo.getDataType().startsWith(DATA_TYPE_RAW_PREFIX));
+            Integer wrapWithInlongMsg = streamInfo.getWrapWithInlongMsg();
+            kafkaSource.setWrapWithInlongMsg(null == wrapWithInlongMsg || 1 == wrapWithInlongMsg);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java:120]: `1` is a magic number. Please extract it as a global constant.

**Review by leezng — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json [hunk 3]
```diff
@@ -374,6 +374,8 @@
   "meta.Stream.ExecuteWorkflow": "执行工作流",
   "meta.Stream.ExecuteConfirm": "确认执行工作流吗?",
   "meta.Stream.ExecuteSuccess": "执行成功",
+  "meta.Stream.WrapWithInlongMsg": "InlongMsg打包",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:377]:
```suggestion
  "meta.Stream.WrapWithInlongMsg": "InlongMsg 打包",
```

**Review by dockerzhang — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json [hunk 4]
```diff
@@ -374,6 +374,8 @@
   "meta.Stream.ExecuteWorkflow": "执行工作流",
   "meta.Stream.ExecuteConfirm": "确认执行工作流吗?",
   "meta.Stream.ExecuteSuccess": "执行成功",
+  "meta.Stream.WrapWithInlongMsg": "InlongMsg 打包",
+  "meta.Stream.WrapWithInlongMsgHelp": "消息体使用InlongMsg打包",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:378]:
```suggestion
  "meta.Stream.WrapWithInlongMsgHelp": "消息体使用 InlongMsg 打包",
```

**Review by healchow — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**